src/main/java/org/kuali/kfs/module/purap/document/web/struts/PurchasingActionBase.java

```diff
@@ -130,8 +130,8 @@
 public class PurchasingActionBase extends PurchasingAccountsPayableActionBase {
 
     private static final Logger LOG = LogManager.getLogger();
-    
-    private static final int SIZE_5MB =5242880;
+
+    private static final int SIZE_15MB = 15728640;

```

Comment:
I see that this max-size check is a Cornell customization. I strongly recommend that you make this limit configurable via a parameter, as opposed to leaving it hard-coded like before. That will make it much easier to reconfigure the limit if we decide to change it in the future.

I would suggest creating a new parameter that is similar to the `MAX_FILE_SIZE_ATTACHMENT` parameter from base code.